app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def index
+    render :json => {test: "Works!!!"}
```

Comment:
use single quoted strings when you are not interpolating. Also, leave spaces before the braces 😄